library/CHART-TEMPLATE/templates/deployment.yaml

```diff
@@ -23,12 +23,16 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "common.serviceAccount.name" . }}
+      {{- with include "common.securityContext.pod" . }}
       securityContext:
-        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+        {{- . | nindent 8 }}
```

Comment:
would make sense to be possible to add the `securityContext:` key inside the helper ? so in the actual template you just need to add the helper entry?